webrender/tests/angle_shader_validation.rs

```diff
@@ -43,7 +44,7 @@ const SHADERS: &[Shader] = &[
     // Cache shaders
     Shader {
         name: "cs_blur",
-        features: CACHE_FEATURES,
+        features: &[ "ALPHA_TARGET", "COLOR_TARGET" ],
```

Comment:
don't we need the same for `cs_border_segment`?

Reply:
No, because it is a LazilyCompiledShader and has no features at all: https://github.com/servo/webrender/blob/master/webrender/src/shade.rs#L629
These two are specified explicitly: 
https://github.com/servo/webrender/blob/master/webrender/src/shade.rs#L501
https://github.com/servo/webrender/blob/master/webrender/src/shade.rs#L509